internal/database/database_container.go

```
@@ -295,58 +380,92 @@ func (s *Container) NewInstance() *InstanceBuilder {
 
-	// Generate the database name and password:
-	name := fmt.Sprintf("test_%d", s.count)
-	user := fmt.Sprintf("test_%d", s.count)
-	password := uuid.NewString()
+	// Generate the database name:
 	s.count++
+	name := fmt.Sprintf("d%d", s.count)
+
+	if b.version != nil {
+		// Slow path: create a blank database and run migrations up to the requested version.
+		_, execErr := s.handle.Exec(fmt.Sprintf(
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).Exec must not be called. use (*database/sql.DB).ExecContext

(noctx)